setupckern.F90

```diff
@@ -501,6 +501,9 @@ subroutine setupckern(carma, cstate, rc)
 
                 ! Now combine all the coagulation and collection kernels into the
                 ! overall kernel.
+                if (.not. do_aer_cld_interact)then
+		  ccd = 0._f
+                end if
```

Comment:
Rather than override the value of `ccd` here, you could change the if block at line 322 to 
```
  if (do_aer_cld_interact) then
     if( re_larg .lt. 1._f )then
        ccd = 0.45_f*cbr*pe3
     else
        ccd = 0.45_f*cbr*pe3*re_larg**(ONE/6._f)
     endif
  else
     ccd = 0._f
  end if
```

Reply:
If I want to implement your suggestions, do I change my branch and push it again? Would that override this ccd branch to reflect your suggestions?

Reply:
That's right.  Make the change in your local copy, commit the change to your branch, and push the branch up to your github fork (again).  Then I will see the updated changes in this PR.  Make sense?